src/Services/UserService.php

```diff
@@ -137,6 +137,69 @@ public function getUserByUUID($uuid)
         return $user;
     }
 
+    /**
+     * Retrieves a single user that is authorized for the calendar
+     * @param $userId
+     * @return array|null
+     */
+    public function getUserForCalendar($userId)
+    {
+        // TODO: eventually we'd like to leverage the inner search piece here and combine these methods
+        return $this->searchUsersForCalendar("", $userId);
+    }
+
+    /**
+     * Retrieves all the users that have been set to show up on the calendar optionally filtered by the facility if one
+     * is provided.
+     * @param string $facility
+     * @return array|null
+     */
+    public function getUsersForCalender($facility = "")
```

Comment:
minor spelling of Calender vs Calendar but is consistent above

Reply:
Thanks for the catch!